mantidproject/mantid
# ILL powder calibration, fix missing scaling

Fixes a missing `Scale` call in the `PowderDiffILLDetEffCorr` algorithm.

**To test:**
Use the `ILL/D20/967076.nxs` from system tests.

```
PowderDiffILLDetEffCorr(CalibrationRun='967076.nxs', OutputWorkspace='calib1')
SaveNexusProcessed(InputWorkspace='calib1', Filename='calibration1.nxs')
PowderDiffILLDetEffCorr(CalibrationRun='967076.nxs', CalibrationFile='calibration1.nxs', OutputWorkspace='calib2')
```

Make sure `calib2` has only 1s.



Fixes #21761 . 

**Release Notes** 


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
